core/storage/feedback/gae_models.py

```diff
@@ -179,21 +251,44 @@ def create(cls, exploration_id, thread_id, message_id):
 
     @classmethod
     def get(cls, exploration_id, thread_id, message_id, strict=True):
-        """Gets the FeedbackMessageModel entry for the given ID.
-
-        If the message id is valid and it is not marked as deleted, returns the
-        message instance. Otherwise:
-        - if strict is True, raises EntityNotFoundError
-        - if strict is False, returns None.
+        """Gets the FeedbackMessageModel entry for the given ID. Raises error if
+        no undeleted messages are found and strict == True.
```

Comment:
This seems incorrect. Shouldn't it be "Raises an error if no undeleted message with the given ID is found, and strict == True."?